aws/templates/common.ftl

```diff
@@ -621,6 +621,18 @@
                         }
                     ]
                 },
+                { 
+                    "Name" : "Website",
+                    "Children" : [
+                        {
+                            "Name": "Index"
```

Comment:
Thinking we should put default here rather than on the macro. Given that we will be using this definition to build the "model", better that the defaults are here. 

Can leave on macro and add here as well but thinking the macro ust expect te index to be provided and the default here provides the default value